### ISSUE TITLE
Light / Dark Mode Color Audit

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9175]
Compare the differences between light and dark modes for accessibility. For example, in light mode, the card background colors are white on a very light gray background, while in dark mode, they are a medium gray on an almost black background. This results in more contrast between the background and the card in dark mode than what we have in light mode, which is not a 1:1 and not consistent.

AC:
- [x] Explore and make recommendations for color updates to create a consistent experience between light and dark mode
     - [x] Include a color contrast comparison
- [x] Explore and make recommendations for components that do not have a dark mode variant (like the tag component)

**Comment by brea11y** [comment]
Findings can be found in the table below or [in the Google sheet](https://docs.google.com/spreadsheets/d/1S7TmyZhms8emh4F5gkG9ivlHYuDsACQP-an8zdpcTWg/edit?usp=sharing) or in the PDF linked below.

[Light _ Dark Mode Component Audit _ VA Mobile App _ September 2024 - Sheet1.pdf](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeVRMQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--1e6e7ea0c2d963dd12f9699f3b8672e79d79b9c3/Light%20_%20Dark%20Mode%20Component%20Audit%20_%20VA%20Mobile%20App%20_%20September%202024%20-%20Sheet1.pdf)
